lib/omnisharp-atom/features/find-symbols.ts

```diff
@@ -0,0 +1,38 @@
+import Omni = require('../../omni-sharp-server/omni')
+import ClientManager = require('../../omni-sharp-server/client-manager');
+import OmniSharpAtom = require('../omnisharp-atom')
+import SpacePen = require('atom-space-pen-views');
+import FindSymbolsView = require('../views/find-symbols-view');
+
+interface TemporaryCodeAction {
+    Name : string;
+    Id: number;
+}
+
+
+class FindSymbols {
+
+    private view : FindSymbolsView;
+    private editor : Atom.TextEditor;
+
+    constructor(private atomSharper: typeof OmniSharpAtom) {
+        this.atomSharper = atomSharper;
+    }
+
+    public activate() {
+        this.atomSharper.addCommand("omnisharp-atom:find-symbols", () => {
```

Comment:
Use OmniSharpAtom.addCommand instead. It handles making sure that the command is only fired on C# files.